### ISSUE TITLE
Update automated test suite to use a test matrix

### ISSUE THREAD

**Issue by cs-cordero** [silviogutierrez/reactivated#18]
We can do this to support older Python versions (I'm not suggesting we support 2.7 😆)  Perhaps as old as 3.6.x, maybe 3.5.x.  Also Django 2.x?

Potential Tooling:
* tox:  https://tox.readthedocs.io/en/latest/
* nox:  https://nox.thea.codes/en/stable/
* pyinvoke:  https://www.pyinvoke.org/
* Others?

Examples in the wild:
dateutil uses tox:  https://github.com/dateutil/dateutil/blob/master/tox.ini
Django uses tox:  https://github.com/django/django/blob/master/tox.ini

**Comment by iorlandini** [comment]
I agree. Not only for the previous ones, that can be easily tested to check which ones are supported out-of-the-box. But for the future ones, keeping support for the current ones.

I vote for tox.
